pom.xml

```diff
@@ -83,7 +83,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <java.level>7</java.level>
+    <java.level>8</java.level>
```

Comment:
Requires a decision by Stapler maintainers. LGTM, but may require extra backporting methods if somebody wants to just update LTS version to the Master branch. I don't